fixpy/fixed_point.py

```diff
@@ -124,29 +114,48 @@ def has_negative_part(array):
 
         return max_bits
 
-
     def __getitem__(self, _slice):
-
         sliced_data = self.data[_slice]
-        return self.__class__(sliced_data,
-                fractional_bits=self.fractional_bits,
-                data_already_scaled=True)
+        return self.__class__(sliced_data, fractional_bits=self.fractional_bits, data_already_scaled=True)
```

Comment:
Quick FYI, the long lines break [pep8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length). Not a big deal, but a nice to have.